src/modules/populateList.js

```diff
@@ -60,13 +62,13 @@ document.querySelector('.add-task .btn').addEventListener('click', () => {
   }
 });
 
-document.querySelector('button[type="reset"]').addEventListener('click', () => {
-  tasks = tasks.filter((task) => !task.completed);
-  tasks.forEach((task, index) => {
-    task.index = index;
+document.querySelector('#reset').addEventListener('click', () => {
+  const updatedTasks = tasks.filter((task) => !task.completed);
+  updatedTasks.forEach((task, index) => {
+    task.index = index + 1;
   });
-  saveTasks();
-  populateList();
+  localStorage.setItem('tasks', JSON.stringify(updatedTasks));
+  document.location.reload();
```

Comment:
- [ ] [OPTIONAL] When I try to edit a todo I get the following error please make sure to fix it so it doesn't confuse the user
<img width="1029" alt="Screenshot 2023-05-05 at 12 12 40" src="https://user-images.githubusercontent.com/61790587/236443665-a3c5c085-48ae-4daf-844b-5ac9234a7225.png">